layouts/partials/location/main.njk

```diff
@@ -0,0 +1,58 @@
+<article
+  class="grav-o-container"
+  typeof="schema:Place"
+  resource="https://buildit.wiprodigital.com/thing/studio/{{ countryCode}}/{{ citySlug }}"
+>
+  <header class="grav-c-page-intro">
+    <h1 property="schema:name">{{ title }} Studio</h1>
+    {% if description %}
+    <p>{{ description }}</p>
+    {% endif %}
+    {% if image %}
+      <img src="{{ image.src }}" alt="{{ image.alt }}" />
+    {% endif %}
+    <link rel="schema:mainEntityOfPage" href="{{ path }}">
+  </header>
+  {# Convoluted way of checking whether contents is blank
+     - i.e. only whitespace #}
+  {% if contents.length > 0 %}
+  <div class="grav-c-ostentatious-copy">
+      {{ contents | safe }}
+  </div>
+  <hr>
+  {% endif %}
+
+  <div class="grav-o-two-column">
+    <section class="grav-c-location-card">
+      <h2>Address</h2>
+      {% include 'partials/location/address.njk' %}
+      <a href="{{ map }}" rel="schema:hasMap" title="See {{ title }} office on Google Maps - Opens in a new tab" target="_blank">
+        <img src="{{ img.url }}" alt="{{ img.alt }}"><br>
+        View map
+      </a>
+    </section>
+
+    <section>
+      <h2>Jobs</h2>
+      {% if jobsByCity[title].length > 0 %}
+      <p>Current job postings for Buildit's {{ title }} studio:</p>
+      <ul>
+        {% for job in jobsByCity[title] %}
+          {% include 'partials/jobs/job-card.njk' %}
+        {% endfor %}
+      </ul>
+      {% set jobLocation = jobsByCity[title][0].location %}
+      <span hidden typeof="schema:Place" resource="{{jobLocation.uri}}">
+        <span property="schema:address" typeof="schema:PostalAddress">
+          <meta property="schema:addressLocality" content="{{jobLocation.city}}">
+          <meta property="schema:addressCountry" content="{{jobLocation.countryCode}}">
+        </span>
+      </span>
+      <p>For open positions in other Buildit studios and locations, please visit our <a href="/jobs/">jobs page</a>.</p>
+      {% else %}
+      <p>Currently, there are no open job postings for Buildit's {{ title }} studio. You can see open positions in other Buildit studios and locations on our <a href="/jobs/">jobs page</a>.</p>
+      {% endif %}
+      <p>If you have any questions please reach out to us at: <a href="mailto:talent-buildit@wipro.com">talent-buildit@wipro.com</a>.</p>
+    </section>
+  </div>
+</article>
```

Comment:
I like the overall look of this template when it has copy and imagery in it - like on the Dublin page. It looks a little less appealing on pages without an image at least. I wonder what would happen if we used the static google map as an image initially?

My other thought is that the Jobs section seems a little text heavy - i wonder if all the copy is needed.

It also seems a bit weird for me that this is two-columned with the address information. This is probably just because it seems to be quite dense with text.

To be fair, everything looks better on the Dublin studio (still not 100% sure we need the level of copy in the Jobs section). So if we can get an image at least then that should help. I've created a mockup of a location page with image, but without copy (notice i've removed the notch and hr):
![localhost_8080_locations_dublin_](https://user-images.githubusercontent.com/46753034/60014473-09bb7480-9679-11e9-9285-4c18d89f1703.png)